Source/Core/InputCommon/ControllerInterface/WGInput/WGInput.h

```diff
@@ -0,0 +1,222 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#pragma warning(push)
+#pragma warning(disable : 4265)
+#include <wrl.h>
+#pragma warning(pop)
+
+#include <windows.gaming.input.h>
+
+#include "InputCommon/ControllerInterface/ControllerInterface.h"
+
+namespace ciface
+{
+namespace WGInput
+{
+namespace WGI = ABI::Windows::Gaming::Input;
+using Microsoft::WRL::ComPtr;
```

Comment:
No "using" in header files please, they get propagated to all units that include this .h.

Reply:
Whats wrong with `using ComPtr = Microsoft::WRL::ComPtr;` ?